watertap/data/techno_economic/nanofiltration.yaml

```diff
@@ -108,29 +108,29 @@ rHGO_dye_rejection:
     cost_method: cost_membrane
     cost_factor: None
     membrane_cost:
-      value: 15
+      value: 50
       units: USD_2018/m^2
     membrane_replacement_rate:
-      value: 0.15
+      value: 0.2
       units: 1/year
   recovery_frac_mass_H2O:
-    value: 0.75
+    value: 0.83
     units: dimensionless
   default_removal_frac_mass_comp:
     value: 0
     units: dimensionless
   removal_frac_mass_comp:
     dye:
-      value: 0.9839
+      value: 0.99
       units: dimensionless
       constituent_longform: Dye
     tds:
-      value: 0.2655
+      value: 0.05
```

Comment:
These are mass removal fractions--not exactly the same as rejection, which we're considering as 1- permeate concentration/feed concentration (and is stored in `rejection_comp`). You can check to see what the newly calculated rejection would be based on these changes and should find that they are higher than these values. Thus, we should change these values back.

Reply:
Nevermind this comment. I realized after the fact that the collaborators specified this update in values, but we should verify that they truly intended these values for mass removal fractions (I think it is possible that they confused these with solute rejection.)

So, no need to revert these values.